### PR TITLE
terraform-provider-kubevirt: e2e test changes

### PR DESCRIPTION
**Pull request by nirarg**

### CODE REVIEW

**Comment by nirarg** [comment]
Hi @brianmcarey 
Do you think it can be merged?
I need this change in order to have e2e run successfully in this PR:
https://github.com/kubevirt/terraform-provider-kubevirt/pull/14

**Comment by brianmcarey** [comment]
Hi @nirarg - sure I think we can merge this. I was trying to see if I could get a rehearsal run first but there seems to be an issue with the rehearsal plugin. 

The existing lane is broken as it is so I think we can merge.

File: github/ci/prow-deploy/files/jobs/kubevirt/terraform-provider-kubevirt/terraform-provider-kubevirt-presubmits.yaml [hunk 1]
```diff
@@ -1,45 +1,41 @@
...
-      fork-per-release: "true"
-      testgrid-create-test-group: "false"
+  - name: pull-terraform-provider-kubevirt-e2e
     always_run: true
-    optional: false
-    skip_report: false
-    cluster: prow-workloads
+    optional: true
     decorate: true
+    skip_report: true
```
Comment (brianmcarey) [inline github/ci/prow-deploy/files/jobs/kubevirt/terraform-provider-kubevirt/terraform-provider-kubevirt-presubmits.yaml:7]: if you want to see feedback on PRs - this needs to be set to false.

**Review by brianmcarey — COMMENTED** [review]
Looks good - just one comment inline. 

This is also why I didn't see the rehearsals. 

Theres a failure in the rehearsal - is that expected? 
https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_project-infra/2626/rehearsal-pull-terraform-provider-kubevirt-e2e/1627607026731323392

File: github/ci/prow-deploy/files/jobs/kubevirt/terraform-provider-kubevirt/terraform-provider-kubevirt-presubmits.yaml (hunk 1 above)
Comment (nirarg) [reply github/ci/prow-deploy/files/jobs/kubevirt/terraform-provider-kubevirt/terraform-provider-kubevirt-presubmits.yaml]: Thanks, fixed

**Comment by nirarg** [comment]
> Theres a failure in the rehearsal - is that expected?
> https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_project-infra/2626/rehearsal-pull-terraform-provider-kubevirt-e2e/1627607026731323392

I'm not sure what is the reason for this failure
It might be that the terraform-provider-kubevirt main branch still using very old `cluster-up` version
I would like to test it with [this PR](https://github.com/kubevirt/terraform-provider-kubevirt/pull/14)

**Review by brianmcarey — APPROVED** [review]
/lgtm

/approve

The existing lane is broken so this update should help testing

**Comment by nirarg** [comment]
Thanks @brianmcarey

**Pull request merged**